cubic-server/Server.hpp

```diff
@@ -45,10 +46,20 @@ class Server
 
     void stop();
 
+    void enforceWhitelistOnReload();
+
     const Configuration::ConfigHandler &getConfig() const {
         return _config;
     }
 
+    WhitelistHandling::Whitelist getWhitelist() const {
```

Comment:
This is strange that we can't use a const ref here but it's ok

Reply:
You know it ends up crashing if we make it const.

Reply:
It's because `isPlayerWhitelisted` is not const

Reply:
Yes, because it can't be because of the iterator, it crashes if it is const

Reply:
And thats why you use auto on iterator.

Try with `const_iterator` 😉

Reply:
Or just use auto

Reply:
Fixed 👍